tasks/pre.yml

```diff
@@ -66,17 +66,66 @@
   tags:
     - fact
 
-- name: redhat family epel installation
+- name: RedHat family (Centos etc..) EPEL installation
   become: 'yes'
   dnf:
     name: "epel-release"
     state: present
     update_cache: 'yes'
-  when: ansible_os_family == "RedHat" and (not ansible_distribution == "Fedora")
+  when: ansible_os_family == "RedHat" and (not (ansible_distribution == "Fedora" or ansible_distribution == "RedHat"))
   tags:
     - dnf
     - packages
 
+- name: RHEL8 repo tasks
+  block:
+    - name: Install RHEL8 EPEL GPG key
+      become: 'yes'
+      ansible.builtin.rpm_key:
+        state: present
+        key: http://download.fedoraproject.org/pub/epel/RPM-GPG-KEY-EPEL-8
+        fingerprint: 94E2 79EB 8D8F 25B2 1810 ADF1 21EA 45AB 2F86 D6A1
```

Comment:
this and the fedora key is public so the public keys can be added to https://github.com/konstruktoid/ansible-role-hardening/blob/master/defaults/main/misc.yml#L4 instead.

Reply:
As you can see, here are two blocks, one for RHEL7 and second for RHEL8. Each use different repo and need different key.. To do not install unnecessary key to the system i choosed this way of the key installation - only when necessary.

If it is not acceptable, i can change this....

Reply:
yeah, I was thinking more about managing this when/if the keys change, it's easier with one variable (which you can change on runtime) instead of remembering specific task changes.

Reply:
So, move it to key(s) list or can it be leaved here? I do not think that this key will be changed in the futurre. Maybe only if will be compromised...

Reply:
Leave it, it will come back to haunt us in a few years ;)